using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -380,6 +615,10 @@ final TargetParameters targetParameters,
 final final AdobeCallback<String> callback);
 ```
 
+* _targetPrefetchList_ is a list of `TargetPrefetch` objects for various mbox locations.
+* _targetParameters_ is the configured `TargetParameters` for the prefetch request.
+* _callback_ is the callback invoked with a `null` value if prefetch is successful, otherwise an error message is returned.
+
```

Comment:
Please reword the bulleted list as follows:
"* *targetPrefetchList* is a list of `TargetPrefetch` objects for various mbox locations.
* *targetParameters* is the configured `TargetParameters` for the prefetch request.
* If the prefetch is successful, *callback* is invoked with a `null` value. If the prefetch is not successful, an error message is returned.